pkgs/applications/networking/cluster/qbec/default.nix

```diff
@@ -0,0 +1,25 @@
+{ lib, buildGoModule, fetchFromGitHub, ... }:
+
+let version = "0.7.5"; in
```

Comment:
Can you use `buildGoModule rec` so it can self reference? Then we can move `version` into the body of the expression.